tools/tidb-binlog-kafka.md

```diff
@@ -117,52 +127,71 @@ zookeeper_addrs = "192.168.0.11:2181,192.168.0.12:2181,192.168.0.13:2181"
-    -data-dir string
-        the path of storing Pump data
-    -kafka-addrs string
-        the connected Kafka address (default "127.0.0.1:9092")
-    -zookeeper-addrs string
-        the Zookeeper address; if you set the option, the Kafka address is got from Zookeeper; if you do not set the option, the value of kafka-addrs is used
-    -gc int
-        the maximum days that the binlog is retained (default 7), and 0 means retaining the binlog permanently
-    -heartbeat-interval int
-        the interval between heartbeats that Pump sends to PD (unit: second)
-    -log-file string
-        the path of the log file
-    -log-rotate string
-        the log file rotating frequency (hour/day)
-    -metrics-addr string
-        the Prometheus pushgataway address; leaving it empty disables Prometheus push
-    -metrics-interval int
-        the frequency of reporting monitoring information (default 15, unit: second)
-    -pd-urls string
-        the node address of the PD cluster (default "http://127.0.0.1:2379")
-    -socket string
-        the monitoring address of the unix socket service (default "unix:///tmp/pump.sock")
+-L string
+    log level: debug, info, warn, error, fatal (default "info")
+-V
+    to print Pump version info
+-addr string
+    the RPC address that Pump provides service (-addr= "192.168.0.10:8250")
+-advertise-addr string
+    the RPC address that Pump provides external service (-advertise-addr="192.168.0.10:8250")
+-config string
+    to configure the file path of Pump; if you specifies the configuration file, Pump reads the configuration first; if the corresponding configuration also exists in the command line argument, Pump uses the command line configuration to cover that in the configuration file
+-data-dir string
+    the path of storing Pump data
+-zookeeper-addrs string (-zookeeper_addrs="192.168.0.11:2181,192.168.0.12:2181,192.168.0.13:2181")
+    the ZooKeeper address; this option gets the Kafka address from ZooKeeper
+-gc int
+    the maximum days that the binlog is retained (default 7), and 0 means retaining the binlog permanently
+-heartbeat-interval int
+    the interval between heartbeats that Pump sends to PD (unit: second)
+-log-file string
+    the path of the log file
+-log-rotate string
+    the log file rotating frequency (hour/day)
+-metrics-addr string
+    the Prometheus pushgataway address; leaving it empty disables Prometheus push
+-metrics-interval int
+    the frequency of reporting monitoring information (default 15, unit: second)
+-pd-urls string
+    the node address of the PD cluster (-pd-urls="http://192.168.0.16:2379,http://192.168.0.15:2379,http://192.168.0.14:2379")
+-socket string
+    the monitoring address of the unix socket service (default "unix:///tmp/pump.sock")
```

Comment:
Indent error from line 155-184. This leads to display chaos. 
Please indent each line of command line option name with four whitespaces, and another four more whitespaces for the explanation line.